js/src/components/google-combo-account-card/connected-google-combo-account-card.scss

```diff
@@ -4,10 +4,18 @@
 		margin: 0;
 	}
 
+	.components-select-control__input {
```

Comment:
@ankitrox We won't need this if we use this approach:
https://github.com/woocommerce/google-listings-and-ads/pull/2639/files#diff-c71501f6145dde774344794f6079c4d6c2f7199047c34fd0eba5484b220bde42R59

Reply:
Removed it.